app/schemas.py

```diff
@@ -758,6 +758,29 @@ def validate_archived(self, data, **kwargs):
             raise ValidationError("Template has been deleted", "template")
 
 
+class ServiceJoinRequestSchema(BaseSchema):
```

Comment:
thank goodness, I did not enjoy marshmallow but since most of the endpoints on the file were using it I was not sure if I should default to json schema or keep going with it.

No worries at all

Reply:
https://github.com/alphagov/notifications-api/pull/4176/commits/99f1fc9c88aac2ba0449d9699a86cf104a6fb35d